### PR TITLE
added article

### PR DESCRIPTION
**Pull request by silchyn**
[DEMO](https://silchyn.github.io/react_article/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -12,6 +13,6 @@ export const App = () => (
   <div className="App">
     <h1>React Article</h1>
 
-    {/* Add an <Article /> and pass `title`, `text` and `date` from the `article` object as props */}
+    <Article title={article.title} date={article.date} text={article.text} />
```
Comment (TwoAndMore) [inline src/App.jsx:16]: [CODE STYLE] - If you have < 3 attribues on a tag and the values are short write the tag in one line (to make it easier to write and read).
```suggestion
    <Article 
      title={article.title} 
      date={article.date} 
      text={article.text}
    />
```

**Review by TwoAndMore — COMMENTED** [review]
Good job

File: src/App.jsx (hunk 1 above)
Comment (ivan-kadykalo) [reply src/App.jsx:16]: Agree. Need to fix

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]

**Review by ivan-kadykalo — APPROVED** [review]
GJ🚀

**Pull request closed without merging**